signalfx-java/src/main/java/com/signalfx/signalflow/SignalFlowClient.java

```diff
@@ -0,0 +1,197 @@
+package com.signalfx.signalflow;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * SignalFx SignalFlow client.
+ *
+ * Client for SignalFx's SignalFlow real-time analytics API. Allows for the execution of ad-hoc
+ * computations, returning its output in real-time as it is produced; to start new background
+ * computations; attach, keep alive or stop existing computations.
+ */
+public class SignalFlowClient {
+
+    private SignalFlowTransport transport;
+
+    /**
+     * Client Constructor that uses default transport/settings
+     * 
+     * @param token
+     *            - user api token
```

Comment:
Don't include that `-` in your parameter documentation (everywhere). This is not how Javadoc formats your stuff.